downloads/github-git-advanced-cheat-sheet.md

```diff
@@ -0,0 +1,45 @@
+# Advanced Cheat Sheet
+
+
+## Merge commands
+
+Abort the merge
+
+```$ git merge --abort```
+
+## Stash commands
+
+Save a stash with a particular name
+
+```$ git stash save <stash name>```
+
+Delete a particular stash
+
+```$ git stash pop <stash name>```
```

Comment:
I think if we're wanting to delete a stash here, we should use `git stash drop`. otherwise `pop` will also apply it